portfolio.css

```diff
@@ -641,3 +732,26 @@ form > ul {
     margin-top: 100px;
   }
 }
+
+.mobile-menu {
+  background: #6070ff;
+  opacity: 0.888;
+  height: 100%;
+  width: 100%;
+  z-index: 9;
+  position: fixed;
+}
```

Comment:
- [ ] As requested by the previous reviewer, kindly ensure that you use the correct margins and padding in your design. This way your design will match the chosen Figma design.

**This is how I am seeing your design right now:**

![image](https://user-images.githubusercontent.com/86318284/195131588-eec510d2-b039-44aa-be43-de575d98a716.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/195131835-255e2a82-c40c-4ff0-9624-696c9e26ab33.png)